recommended-rules.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: contribute
+title: Recommended Rules
+---
+
+# Recommended Rules
+
+In the documentation, we provide a list of
+[recommended rules](https://docs.bazel.build/versions/master/rules.html).
+
+This is a set of high quality rules, which will provide a good experience to our
+users. We make a distinction between the supported rules, and the hundreds of
+rules you can find on the Internet.
+
+## Nomination
+
+If a rule meets the requirements below, a rule maintainer can nominate it to be
```

Comment:
rule or ruleset? rule means a single foo_binary() rule to me, whereas ruleset means foo_binary, foo_library, etc.

Reply:
Updated to ruleset